pkgs/by-name/as/asusctl/package.nix

```diff
@@ -93,6 +94,8 @@ rustPlatform.buildRustPackage rec {
       --add-needed ${lib.getLib libxkbcommon}/lib/libxkbcommon.so.0
   '';
 
+  passthru.updateScript = nix-update-script { };
```

Comment:
Isn't really necessary, nixpkgs-update can handle things fine without it.

Reply:
We shall really mention this in CONTRIBUTING.md, an increasing number of these are popping up.

Reply:
This is something I as a beginner also recently stumbled upon. It is not clear at all, what is the difference between adding the script and not adding it.